k8-util/helm/fluvio-sys/templates/crd_partition.yaml

```diff
@@ -20,14 +20,39 @@ spec:
           properties:
             spec:
               type: object
-              required: ["leader", "replicas"]
+              required: ["leader"]
               properties:
                 leader:
                   type: integer
                 replicas:
                   type: array
                   items:
                     type: integer
+                mirror:
+                  type: object
+                  oneOf:
+                    - required: ["target"]
+                    - required: ["source"]
+                  properties:
+                    target:
+                      type: object
+                      required: ["remoteCluster","sourceReplica"]
+                      properties:
+                        sourceReplica:
+                          type: string
+                        remoteCluster:
+                          type: string
+                    source:
+                      type: object
+                      required: ["upstreamCluster","targetSpu"]
```

Comment:
changed this to `homeCluster` to be consistent with new cmd name

Reply:
I was already working on it, but I renamed to core and edge.
I thought that home and remote would be only a syntax cmd think for users.

Rename thinks to remote now would be a little confuse because remote is an enum variant that could be an edge or core.

If makes sense to change all code to home and remote, I will need to change previous pull request to it too.

What do you think?

Reply:
We decided rename to home and remote. 
Fixed!